### PR TITLE
This puppet module currently does the following:

### PR DESCRIPTION
**Pull request by usnavi**
- ensures that flume-repose & cf-flume-sink rpms exist or removes them
- creates the /etc/sysconfig/flume-ng
- contains spec tests

One note:  This module does not install any rpms, unlike puppet-repose.
When I approached @gregswift with a question on how to incorporate 2 distinct
version numbers into a single puppet module, he mentioned that he actually
prefers to manage it outside of puppet.  I was fine wtih that as eventually
the Ops group will be deploying this with other groups.

Future PR's will flesh out the actual flume configuration.

@gregswift - what's the best way to test this?  Can we configure my inova
to pull down this conf and I can verify that it works?

### CODE REVIEW

**Comment by gregswift** [comment]
So.. just to clarify.  I do think that this should ensure the packages are installed. What i'm not a fan of is adding lots of complexity to pin the versions.

As for the best way to test, we can have it available in the dev environment pretty readily.  But you can also just apply it locally on your inova instance.  Can make sure you get some help w/ that tomorrow.

**Comment by usnavi** [comment]
@gregswift - I just updated this PR with the following:

First attempt on implementing a puppet module to configure the following
- install flume-repose
- install cf-flume-sink
- configure flume-ng service
- configure flume_env.sh for java args
- configure log4j.properities to enable log rotation & syslog
- configure cf-flume-conf.properties to configure flume & cf-flume-sink

I haven't implemented any tests for it, but I have been testing on vagrant.  My goal is to get a PR accepted so I can start verifying this with puppet-cloudfeeds & puppet-profile_cloudfeeds.  Future PRs will contain tests.

File: .fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fixtures:
+  repositories:
+    stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
```
Comment (gregswift) [inline .fixtures.yml]: To ensure we are getting the latest release, not the latest from master we should probably reference forge.  This doesn't have to happen right now, but just putting it out there.

```
fixtures:
  forge_modules:
    stdlib: "puppetlabs/stdlib"
```

File: manifests/params.pp [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  }
+  
+  ## daemon_home for valve
+  $daemon_home = '/opt/flume'
+
+  ## pid file for valve
+  $pid_file = '/var/run/flume-ng.pid'
+
+  ## JAVA_HOME
+  $java_home = '/usr/lib/jvm/jre-1.7.0-openjdk.x86_64'
```
Comment (gregswift) [inline manifests/params.pp]: we should never be referencing a versioned path directly. there are standardized symlinks.

```
$java_home = '/usr/lib/jvm/jre'
```

**Comment by usnavi** [comment]
@shintasmith @ChandraAddala @VinnyQ @bjoshua - puppet...in yo face!

**Comment by usnavi** [comment]
@gregswift - updated as per your comments.  If it looks decent enough, please accept and I'll start testing against puppet-cloufeeds.

**Comment by gregswift** [comment]
lgtm

**Pull request merged**